pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+
+import SnapshotCard from '@pkg/components/SnapshotCard.vue';
+
+interface Data {
+  // eslint-disable-next-line no-undef
+  snapshotsPollingInterval: NodeJS.Timer | null,
```

Comment:
Previously, we've used `ReturnType<typeof setInterval> | undefined` to work with both `setInterval` and `clearInterval` 

```suggestion
  snapshotsPollingInterval: ReturnType<typeof setInterval> | undefined;
```